client/src/components/Button.tsx

```diff
@@ -16,25 +15,19 @@ const Container = styled.div`
   padding: 0.5rem;
 `;
 
-function Button({ className, children, primary, secondary }) {
+type Props = {
+  primary?: boolean;
+  secondary?: boolean;
+  children: React.ReactNode;
+  className?: string;
+};
+
+function Button({ className, children, primary, secondary }: Props) {
```

Comment:
if I'm not mistaken, if you were to change Props to an interface or possibly a class, you could shorten this by quite a bit, in fact you might be able to do it without changing it to an interface (will have to look up how type aliases work)

But like is there any reason you can't simply do something like myProps: Props?